ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-e2e-commands.sh

```diff
@@ -0,0 +1,21 @@
+        #Gather tag and image for deployment
+        IN=$ODH_NOTEBOOK_CONTROLLER_IMAGE
+        arrIN=(${IN//:/ })
+        export IMG=${arrIN[1]}  
+        export TAG=${arrIN[2]}
+
+        # Setup and test odh-nbc
+        pushd components/odh-notebook-controller
+        oc new-project odh-notebook-controller-system
+        make deploy -e K8S_NAMESPACE=odh-notebook-controller-system
+
+        mkdir -p ~/.kube
+        cp /tmp/kubeconfig ~/.kube/config 2> /dev/null || cp /var/run/secrets/ci.openshift.io/multi-stage/kubeconfig ~/.kube/config
+        chmod 644 ~/.kube/config
+        export KUBECONFIG=~/.kube/config
```

Comment:
We can simplify this block to use `/tmp/kubeconfig` as the KUBECONFIG file and copy `/var/run/secrets/ci.openshift.io/multi-stage/kubeconfig` to that location as the default